it/v2/endpoints/DeleteHistoricNonFhlUkPropertyAnnualSubmissionControllerISpec.scala

```diff
@@ -0,0 +1,139 @@
+/*
+ * Copyright 2022 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package v2.endpoints
+
+import com.github.tomakehurst.wiremock.stubbing.StubMapping
+import play.api.http.HeaderNames.ACCEPT
+import play.api.http.Status
+import play.api.libs.json.{ JsObject, Json }
+import play.api.libs.ws.{ WSRequest, WSResponse }
+import play.api.test.Helpers.AUTHORIZATION
+import support.V2IntegrationBaseSpec
+import v2.models.errors._
+import v2.stubs.{ AuditStub, AuthStub, DownstreamStub, MtdIdLookupStub }
+
+class DeleteHistoricNonFhlUkPropertyAnnualSubmissionControllerISpec extends V2IntegrationBaseSpec {
+
+  private trait Test {
+
+    val nino: String    = "AA123456A"
+    val taxYear: String = "2021-22"
+
+    def uri: String = s"/uk/annual/non-furnished-holiday-lettings/$nino/$taxYear"
+
+    def desUri: String = s"/income-tax/nino/$nino/uk-properties/other/annual-summaries/2022"
```

Comment:
Could we rename desUri to ifsUri to avoid confusion? Tech spec says backend is IFS. Probably worth fixing this in the FHL Controller ISpec too.

Reply:
Nevermind - tech spec was incorrect and backend is in fact des